TicketSwap/solar
# feat(onetimecodeinput): added a OneTimeCodeInput field

# Description

This component is introduced to handle one time code password to, for example, verify a users
phone number. 

## How to test

- Checkout this branch
- `$ yarn storybook`
- Go to `Inputs/OneTimeCodeInput
- Enter a 5 digit code, manually
- Navigate between the fields from left to right, or right to left
- Use backspace to remove an input and confirm the focus moves to the previous field
- Copy and paste a 5 digit code and paste it into the first field, confirms it spreads the code over all input fields
- Copy and paste a code that is longer than 5 digits and confirm it does nothing at all.
- If you really want to go hardcode: get storybook to run on your mobile browser and ask @glenngijsberts if he wants to send you text that goes says: `Code: 68420`. Check if your phone prompts you with the OneTimeCode from the text message

## Screenshots

![image](https://user-images.githubusercontent.com/12168698/141337057-76efe1df-34e3-425a-a0cc-b98d181c06d6.png)
